README.md

```diff
@@ -61,8 +61,8 @@ Once you deploy your app, you can embed this badge right into your GitHub readme
 
 ## Streamlit for Teams
 
-[Streamlit for Teams](https://streamlit.io/for-teams/) is our enterprise solution for deploying, managing, sharing, and collaborating on your Streamlit apps. Streamlit for Teams provides secure single-click deploy, authentication, web editing, versioning, and much more. It is currently in closed beta, and you can [join the waitlist here](https://streamlit.io/for-teams/).
+[Streamlit for Teams](https://streamlit.io/for-teams/) is our enterprise solution for deploying, managing, sharing, and collaborating on your Streamlit apps. Streamlit for Teams provides secure single-click deploy, authentication, web editing, versioning, and much more. It is currently in closed beta, and you can [join the wait-list here](https://streamlit.io/for-teams/).
```

Comment:
nit, waitlist as one word is more stylistically correct

Reply:
Here they are using as none, not a word.